res/values/strings.xml

```diff
@@ -465,6 +465,35 @@
     <!-- the explanation is shown (1) as a modal dialog with the buttons "Cancel" and "Continue" as well as (2) as a device message -->
     <string name="aeap_explanation">You changed your email address from %1$s to %2$s.\n\nIf you now send a message to a verified group, contacts there will automatically replace the old with your new address.\n\nIt\'s highly advised to set up your old email provider to forward all emails to your new email address. Otherwise you might miss messages of contacts who did not get your new address yet.</string>
 
+
+    <!-- Multi Device -->
+    <!-- "Second Device" can also be translated as "Another Device", if that is catchier in the destination language. However, make sure to use the term consistently. -->
+    <string name="multidevice_title">Add Second Device</string>
+    <string name="multidevice_same_network_hint">Make sure both devices are on the same Wi-Fi or network</string>
+    <string name="multidevice_install_dc_on_other_device">Install and start Delta Chat 1.36 or newer on your other device (https://get.delta.chat)</string>
+    <string name="multidevice_tap_scan_on_other_device">During setup, tap “Scan QR Code” and scan the code shown here</string>
```

Comment:
I don't think users will understand what "During setup" means.

My favorite would be `Install Delta Chat, tap "Scan QR Code" ...`. This 1. hints at the fact that it needs to be done on another device 2. hints at the fact that it has to be done on the screen that turns up after installation.

Reply:
idea is to underline that you have to scan the code from inside delta chat

Reply:
Another possibility would be `Open Delta Chat, ...`, then. I still like `Install Delta Chat, ...` a very small bit better, though.

Reply:
i changed that with the recent commit to:

<img width="283" alt="Screenshot 2023-03-24 at 11 12 52" src="https://user-images.githubusercontent.com/9800740/227493493-a7d0d7ff-0f55-4f96-902a-6edf4d183269.png">